scripts/irods/test/test_istream.py

```diff
@@ -173,6 +174,18 @@ def test_client_side_configured_default_resource_is_honored_on_writes__issue_529
             self.admin.assert_icommand(['iadmin', 'rmresc', resc_name])
             self.user.environment_file_contents = old_env_config
 
+    def test_opening_a_non_existent_data_object_does_not_cause_a_stacktrace_to_be_logged__issue_5419(self):
```

Comment:
This will need to be skipped in topology tests due to reading from the log, I think.

Reply:
added skip decorator.